### DISCUSSION TITLE
Tables not being rendered properly

### DISCUSSION THREAD

**Discussion by zeulewan** [squidfunk/mkdocs-material#6715] [General]
Look at any table rendered by a Mkdocs site using material, and look at one not. Where are the lines separating columns? This seems to be a pretty big issue. Great work on this btw I love material

<img width="289" alt="Screenshot 2024-01-29 at 10 59 47 PM" src="https://github.com/squidfunk/mkdocs-material/assets/55005377/a046593e-f342-4fff-bccc-d51e74d692ee">
<img width="548" alt="Screenshot 2024-01-29 at 11 05 07 PM" src="https://github.com/squidfunk/mkdocs-material/assets/55005377/503b654c-486e-4713-8b66-7c6acd965315">

**Comment by facelessuser** [comment]
It's not an issue, just a stylistic choice. You can use custom CSS to add cell borders if you like.

**Reply by zeulewan** [reply]
got it thanks

**Reply by markallasread** [reply]
And just for reference @zeulewan: You can override CSS like such https://squidfunk.github.io/mkdocs-material/customization/#additional-css 

I think one of the better examples of how to do this is shown  #3406 — it's a bit of a deep dive on the topic, but gives a whole lot of insight into what overriding components of the theme looks like.

**Reply by zeulewan** [reply]
Cool, thanks
